### ISSUE TITLE
collector pod is not scheduling for all nodes

### ISSUE THREAD

**Issue by amrashwan** [eraser-dev/eraser#503]
**What steps did you take and what happened:**
collector pod is not scheduling for all nodes 
Please create a property/settings in the chart to pass the toleration to the collector pod

Example of the errors
52s         Normal    Started                pod/collector-aks-system-18979628-vmss000000-fsghv   Started container trivy-scanner
56s         Warning   TaintToleration        pod/collector-aks-xinfra-11501615-vmss000000-x4r89   Predicate TaintToleration failed
56s         Normal    TaintManagerEviction   pod/collector-aks-xinfra-11501615-vmss000000-x4r89   Marking for deletion Pod xinfra/collector-aks-xinfra-11501615-vmss000000-x4r89
56s         Normal    TaintManagerEviction   pod/collector-aks-xinfra-11501615-vmss000001-htknt   Marking for deletion Pod xinfra/collector-aks-xinfra-11501615-vmss000001-htknt
56s         Warning   TaintToleration        pod/collector-aks-xinfra-11501615-vmss000001-htknt   Predicate TaintToleration failed
56s         Normal    TaintManagerEviction   pod/collector-aks-xinfra-11501615-vmss000002-l2mct   Marking for deletion Pod xinfra/collector-aks-xinfra-11501615-vmss000002-l2mct
56s         Warning   TaintToleration        pod/collector-aks-xinfra-11501615-vmss000002-l2mct   Predicate TaintToleration failed
56s         Normal    TaintManagerEviction   pod/collector-aks-xinfra-11501615-vmss000003-bxsrb   Marking for deletion Pod xinfra/collector-aks-xinfra-11501615-vmss000003-bxsrb
56s         Warning   TaintToleration        pod/collector-aks-xinfra-11501615-vmss000003-bxsrb   Predicate TaintToleration failed
56s         Normal    TaintManagerEviction   pod/collector-aks-xinfra-11501615-vmss000004-tlgwz   Marking for deletion Pod xinfra/collector-aks-xinfra-11501615-vmss000004-tlgwz
56s         Warning   TaintToleration        pod/collector-aks-xinfra-11501615-vmss000004-tlgwz   Predicate TaintToleration failed

**What did you expect to happen:**
To schedule collector pods to all linux nodes on the cluster

**Environment:**

- Eraser version: 0.5.0
- Kubernetes version: (use `kubectl version`): 1.24.6

**Comment by pmengelbert** [comment]
default tolerations to schedule on all nodes:
```
default = [
  {
     effect : "NoExecute",
     operator : "Exists"
  },
  {
      effect : "NoSchedule"
      operator : "Exists"
   },
   {
     effect : "PreferNoSchedule"
     operator : "Exists"
   }
]
```

**Comment by pmengelbert** [comment]
will be fixed by #508

**Comment by pmengelbert** [comment]
fixed by #508
